### PR TITLE
Use eslint-config-stripes

### PR DESCRIPTION
**Pull request by cherewaty**
Switches to using shared FOLIO eslint config: https://github.com/folio-org/eslint-config-stripes

### CODE REVIEW

File: lib/MenuItem/MenuItem.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const handleItemSelect = (e, props) => {
 };
 
 
-const MenuItem = props => (<div onClick={e => handleItemSelect(e, props)}> {props.children}</div>);
+const MenuItem = props => (<div onClick={e => handleItemSelect(e, props)}> {props.children}</div>); // eslint-disable-line jsx-a11y/no-static-element-interactions
```
Comment (cowboyd) [inline lib/MenuItem/MenuItem.js]: Do we need to follow this up with a PR to handle it in it the `jsx-a11y` approved way?

Comment (cherewaty) [reply lib/MenuItem/MenuItem.js]: Yes we do.

Comment (cowboyd) [reply lib/MenuItem/MenuItem.js]: Ok, let's make sure we've got an issue open somwhere. Otherwise, I'm 👍

Comment (JohnC-80) [reply lib/MenuItem/MenuItem.js]: Just a heads-up, I've got a branch 'dropdown-revisions' that resolves the lint issue.. had to do a bit of a surgery there. `<MenuItem>` is a new addition to the new `<Dropdown>` component set. so there's some work-in-progress there as we integrate the component and swap out the old react-bootstrap Dropdown instances...

**Pull request merged**